### PR TITLE
Remove trailing spaces

### PR DESCRIPTION
**Pull request by thinceller**
#### remove trailing spaces

When I was using example I noticed trailing spaces.
So, this PR removes the trailing spaces of json file, README, and others.

`examples/with-jest-typescript/src/modules/cars/Overview.tsx` also has it, but this time it did not change as tslint error occurs at commit.

### CODE REVIEW

File: examples/custom-server-actionhero/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Running Next.JS and React /inside/ of ActionHero
 
-This server will render dynamic next.js/react pages on some routes, and normal ActionHero API requests on others.
```
Comment (Timer) [inline examples/custom-server-actionhero/README.md]: Two spaces in markdown actually means a line break. Please replace this with a `<br>`.

File: examples/custom-server-actionhero/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Running Next.JS and React /inside/ of ActionHero
 
-This server will render dynamic next.js/react pages on some routes, and normal ActionHero API requests on others.  
+This server will render dynamic next.js/react pages on some routes, and normal ActionHero API requests on others.
```
Comment (Timer) [inline examples/custom-server-actionhero/README.md]:
```suggestion
This server will render dynamic next.js/react pages on some routes, and normal ActionHero API requests on others.<br>
```

File: examples/custom-server-actionhero/README.md [hunk 3]
```diff
@@ -100,7 +100,7 @@ module.exports = class CreateChatRoom extends Action {
 
 3. Tell ActionHero to use the api rather than the file server as the top-level route in `api.config.servers.web.rootEndpointType = 'api'`.  This will allows "/" to listen to API requests.  Also update `api.config.general.paths.public = [ path.join(__dirname, '/../static') ]`.  In this configuration, the next 'static' renderer will take priority over the ActionHero 'public file' api.  Note that any static assets (CSS, fonts, etc) will need to be in "./static" rather than "./public".
 
-Note that this is where the websocket server, if you enable it, will place the `ActionheroWebsocketClient` libraray.  
+Note that this is where the websocket server, if you enable it, will place the `ActionheroWebsocketClient` libraray.
```
Comment (Timer) [inline examples/custom-server-actionhero/README.md]:
```suggestion
Note that this is where the websocket server, if you enable it, will place the `ActionheroWebsocketClient` library.<br>
```

File: examples/custom-server-koa/README.md [hunk 4]
```diff
@@ -50,10 +50,10 @@ The example shows a server that serves the component living in `pages/a.js` when
 
 ## Side note: Enabling gzip compression
 
-The most common Koa middleware for handling the gzip compression is [compress](https://github.com/koajs/compress), but unfortunately it is currently not compatible with Next.  
-`koa-compress` handles the compression of the response body by checking `res.body`, which will be empty in the case of the routes handled by Next (because Next sends and ends the response by itself). 
+The most common Koa middleware for handling the gzip compression is [compress](https://github.com/koajs/compress), but unfortunately it is currently not compatible with Next.
```
Comment (Timer) [inline examples/custom-server-koa/README.md]:
```suggestion
The most common Koa middleware for handling the gzip compression is [compress](https://github.com/koajs/compress), but unfortunately it is currently not compatible with Next.<br>
```

File: examples/with-next-less/README.md [hunk 5]
```diff
@@ -41,5 +41,5 @@ now
 
 ## The idea behind the example
 
-This example demonstrates how to use the [next-less plugin](https://github.com/zeit/next-plugins/tree/master/packages/next-less).  
+This example demonstrates how to use the [next-less plugin](https://github.com/zeit/next-plugins/tree/master/packages/next-less).
```
Comment (Timer) [inline examples/with-next-less/README.md]:
```suggestion
This example demonstrates how to use the [next-less plugin](https://github.com/zeit/next-plugins/tree/master/packages/next-less).<br>
```

**Review by Timer — CHANGES_REQUESTED** [review]

**Comment by thinceller** [comment]
@Timer 
I was forgotten to a line break, thanks🙌

So, I am wondering which of spaces and `<br>` is common🤔
`<br>` is redundant, but I prefer it to spaces.

**Comment by Timer** [comment]
Spaces will accidentally be removed like you did here. `<br>` is more explicit. 😄

**Review by timneutkens — APPROVED** [review]

**Pull request merged**